guardian/pinboard
# Update eslint to latest versions

## What does this change?

This PR updates the ESLint tool to the latest versions.

Note that this includes moving to ESLint's new config system - [see relevant posts about this made to the ESLint blog](https://eslint.org/blog/category/api-changes/) .

The primary aim of this update is to get rid of some Dependabot security alerts, and to lower technical debt going forward.

## How to test
+ Run `yarn lint` locally - there should be plenty of warnings, but no errors reported.
+ Run the PR in the CODE environment to see if the new system brakes any CI steps.
+ Test in CODE - everything should operate as before
